pkg/minishift/openshift/openshift.go

```diff
@@ -225,3 +252,53 @@ func deleteBackup(target OpenShiftPatchTarget, id string, commander docker.Docke
 
 	return nil
 }
+
+func getLocalConfigFile(target string) string {
+	openshiftVersion := getOpenshiftVersion()
+	switch target {
+	case "master":
+		valid, _ := openshiftVersionCheck.IsGreaterOrEqualToBaseVersion(openshiftVersion, constants.RefactoredOcVersion)
+		if !valid {
+			return "/var/lib/minishift/openshift.local.config/master/master-config.yaml"
+		}
+		return "/var/lib/minishift/base/openshift-apiserver/master-config.yaml"
+	case "node":
+		valid, _ := openshiftVersionCheck.IsGreaterOrEqualToBaseVersion(openshiftVersion, constants.RefactoredOcVersion)
+		if !valid {
+			return "/var/lib/minishift/openshift.local.config/node-localhost/node-config.yaml"
+		}
+		return "/var/lib/minishift/base/node/node-config.yaml"
+	}
+	return ""
+}
```

Comment:
@praveenkumar  I think `valid, _ := openshiftVersionCheck.IsGreaterOrEqualToBaseVersion(openshiftVersion, constants.RefactoredOcVersion)` can run before starting of the switch case. It will reduce the line of code and improve the readability little bit.  Same comment for below function.

Reply:
```diff
diff --git a/pkg/minishift/openshift/openshift.go b/pkg/minishift/openshift/openshift.go
index 68bb83f7..a612a7ca 100644
--- a/pkg/minishift/openshift/openshift.go
+++ b/pkg/minishift/openshift/openshift.go
@@ -255,15 +255,14 @@ func deleteBackup(target OpenShiftPatchTarget, id string, commander docker.Docke

 func getLocalConfigFile(target string) string {
        openshiftVersion := getOpenshiftVersion()
+       valid, _ := openshiftVersionCheck.IsGreaterOrEqualToBaseVersion(openshiftVersion, constants.RefactoredOcVersion)
        switch target {
        case "master":
-               valid, _ := openshiftVersionCheck.IsGreaterOrEqualToBaseVersion(openshiftVersion, constants.RefactoredOcVersion)
                if !valid {
                        return "/var/lib/minishift/openshift.local.config/master/master-config.yaml"
                }
                return "/var/lib/minishift/base/openshift-apiserver/master-config.yaml"
        case "node":
-               valid, _ := openshiftVersionCheck.IsGreaterOrEqualToBaseVersion(openshiftVersion, constants.RefactoredOcVersion)
                if !valid {
                        return "/var/lib/minishift/openshift.local.config/node-localhost/node-config.yaml"
                }
```